.pre-commit-config.yaml

```diff
@@ -1,9 +1,5 @@
 ---
 files: ^(.*\.(py|yaml))$
-# We need to match the exclude list in pyproject.toml because pre-commit.ci
-# passes filenames and these do not get passed through the tool.black filter
-# for example
-exclude: ^(\.[^/]*cache/.*|.*/freesurfer/contrib/.*)$
```

Comment:
@larsoner I believe we needed this because we had listed the excludes in the wrong place in `pyproject.toml`, but let's see